src/Microsoft.ML.Transforms/Expression/TokKind.cs

```diff
@@ -0,0 +1,212 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Internal.Utilities;
+using Microsoft.ML.Runtime;
+
+namespace Microsoft.ML.Transforms
+{
+    internal enum TokKind
+    {
+        None,
+
+        // Miscellaneous
+        Eof,
+        Error,
+        ErrorInline,
+
+        // Literals
+        IntLit,
+        FltLit,
+        DblLit,
+        CharLit,
+        StrLit,
+
+        // Noise
+        Comment,
+        NewLine,
+
+        // Punctuators
+
+        Add, // +
+        AddAdd,
+        AddEqu,
+        Sub, // -
+        SubSub,
+        SubEqu,
+        SubGrt,
+        Mul, // *
+        MulEqu,
+        Div, // /
+        DivEqu,
+        Per, // %
+        PerEqu,
+
+        Car, // ^
+        CarEqu,
+        Amp, // &
+        AmpAmp,
+        AmpEqu,
+        Bar, // |
+        BarBar,
+        BarEqu,
+
+        Til, // ~
+        Bng, // !
+        BngEqu,
+
+        Equ, // =
+        EquEqu,
+        EquGrt,
+        Lss,
+        LssLss,
+        LssEqu,
+        LssGrt,
+        LssLssEqu,
+        Grt,
+        GrtGrt,
+        GrtEqu,
+        GrtGrtEqu,
+
+        Que, // ?
+        QueQue,
+
+        Dot,
+        Comma,
+        Colon,
+        ColonColon,
+        Semi,
+
+        OpenCurly,
+        OpenParen,
+        OpenSquare,
+
+        CloseCurly,
+        CloseParen,
+        CloseSquare,
+
+        // Words - identifier and key words
+        Ident,
+
+        False,
+        True,
+        Not,
+        And,
+        Or,
+
+        // REVIEW: These are specific to the NetParser. Use a general mechanism.
+        Const,
+        Input,
+        Output,
+        Hidden,
+        Share,
+        Sigmoid,
+        Linear,
+        SoftMax,
+        RectifiedLinear,
+        Square,
+        Sqrt,
+        SoftRectifiedLinear,
+        Tanh,
+        BoundedRectifiedLinear,
+        From,
+        All,
+        Where,
+        Convolve,
```

Comment:
Are the Net# tokens being kept for internal compatibility? Otherwise we could remove the non-used tokens.

Reply:
Resolved in follow-up PR: https://github.com/dotnet/machinelearning/pull/4614/files#r362319272

#Resolved